llvm/unittests/DebugInfo/DWARF/DWARFDebugLineTest.cpp

```diff
@@ -2035,4 +2035,139 @@ TEST_F(DebugLineBasicFixture, PrintPathsProperly) {
   EXPECT_THAT(Result.c_str(), MatchesRegex("a dir.b dir.b file"));
 }
 
+/// Test that lookupAddressRange correctly filters rows based on
```

Comment:
Does this test need any DIEs? Or could it just create a line table and test that in isolation?